### DISCUSSION TITLE
アドベントカレンダーのリマインド通知の日付がずれている

### DISCUSSION THREAD

**Discussion by nanasi-1** [increments/qiita-discussions#930] [要望・アイデア・不具合]
#586 と同じ不具合だと思います。

---

<img width="819" alt="スクリーンショット 2024-11-26 10 39 26" src="https://github.com/user-attachments/assets/aff0c686-d7b2-45df-a866-ef525e3ee96c">

[通知一覧](https://qiita.com/notifications?scope=advent_calendar)からアドベントカレンダーのリマインド通知を見たところ、日付が1日ずれているように感じました。
カレンダーの担当日は早くても12/1ですが、1日前にずれているため11/30と表示されているのだと思います。

なお、ヘッダーの通知アイコンを押すことで見られる通知では、正常な日付（`2024/12/1`、`2024/12/2`）が表示されていました。

**Comment by P-SiZK** [comment]
@nanasi-1 
ご報告ありがとうございます！

こちら https://github.com/increments/qiita-discussions/discussions/586 と同じ原因と思われます。
現在、修正中ですのでお待ちいただけますと幸いです。

**Reply by P-SiZK** [reply]
@nanasi-1 
お待たせいたしました！
リマインド通知の日付がずれてしまう不具合の修正が完了しました！
